README.md

```diff
@@ -0,0 +1,55 @@
+# Asserts Trace Processor
```

Comment:
Will remove the other modules and move this inside.

The need for the `assertsprocessor` folder structure is driven by how the otel collector build process for inclusion of custom components.

Reply:
> I am curious about the file structure of this repo. This README is actually outside of the `assertsprocessor`, so is it going to be the README for all the projects in this repo?

Removed the other modules. Thinking to keep the README at the top level as this will be the only component in this repo now.